### PR TITLE
Update Submission Scripts

### PR DESCRIPTION
**Pull request by nikosbosse**

### CODE REVIEW

File: submissions/submit.sh [hunk 1]
```diff
@@ -1,19 +1,19 @@
-#!bin/bash
+#!/bin/bash
 
 #define date
 ForecastDate=$(date +'%Y-%m-%d')
 
 # Clone the hub repository if not already present
 #git clone --depth 1 https://github.com/epiforecasts/covid19-forecast-hub-europe
-
+git
```
Comment (seabbs) [inline submissions/submit.sh:8]: error

File: submissions/submit.sh [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+git 
 # install GitHub CLI
 # https://cli.github.com/
 
 # Authenticate with GitHub
 # gh auth login
 
 # Update the hub repository
-cd ../covid19-forecast-hub-europe
+cd ../covid19-forecast-hub-europecd
```
Comment (seabbs) [inline submissions/submit.sh:16]: error

**Comment by nikosbosse** [comment]
updated

**Comment by nikosbosse** [comment]
Thanks for spotting!

**Pull request merged**